docs/includes/virtualenv.txt

```diff
@@ -0,0 +1,4 @@
+.. note:: Ensure you are running in a virtual environment, so you can install, upgrade, and       
+          remove packages using the program `pip`. Run the command `python3 -m venv .venv` to create and manage your virtual environments. To begin using the virtual environmen, use the command `source venv/bin/activate` to activate.   
```

Comment:
In restructured text, double backticks (``) are used for inline code formatting, while single backticks italicize. All command references here should be enclosed in double backticks.

Reply:
Typo: "virtual environmen" -> "virtual environment"

Reply:
I would clarify the sentence ending with "using the program `pip`". a little bit. You can install packages with or without `pip`, what a virtualenv provides is a way to isolate the installation of specific dependencies of a specific application, so that the dependencies of another application (or updates to the OS at a whole) do not get in the way of that application's requirements

Reply:
Typo in the command: `source venv/bin/activate` -> `source .venv/bin/activate` (note leading `.`)

Reply:
Nit: "create and manage your virtual environments" -> "create and manage your virtual environment" (in this case you're just creating one, not multiple)